ImperatorToCK3/Data_Files/blankMod/output/common/culture/eras/~00_culture_eras.txt

```diff
@@ -0,0 +1,54 @@
+﻿#culture_era_classical = {
+#	year = 0
+#	modifier = {
+#		domain_limit = 4
+#		tyranny_gain_mult = -0.25
+#		diplomatic_range_mult = -0.75
+#		title_creation_cost_mult = 100 # Title creation impossible before CK3 proper
+#		movement_speed = -0.5
+#	}
+#}
+#culture_era_late_classical = {
+#	year = 312 # Battle of the Milvian Bridge
+#	modifier = {
+#		domain_limit = 0.25
+#		tyranny_gain_mult = 0.5
+#		army_maintenance_mult = -0.5
+#		diplomatic_range_mult = 0.25
+#		movement_speed = 0.25
+#	}
+#}
+#culture_era_tribal = {
+#	year = 535 # Start of Gothic War
+#	modifier = { # Restoring base CK3 stuff
+#		tyranny_gain_mult = -0.25
+#		army_maintenance_mult = 0.5
+#		diplomatic_range_mult = 0.25
+#		title_creation_cost_mult = -100
+#		movement_speed = 0.25
+#	}
+#}

```

Comment:
I guess you need to check which of these modifiers caused the crash. Are that many modifiers needed in the first place?

Reply:
Yes and no, this was just a provisional revision of eras to fit seven more centuries of game. As I will get tech modding, I will see and fix this.